lldb/source/Commands/CommandObjectTarget.cpp

```diff
@@ -2005,9 +2021,10 @@ class CommandObjectTargetModulesDumpSymtab
             result.GetOutputStream().EOL();
             result.GetOutputStream().EOL();
           }
-          if (INTERRUPT_REQUESTED(GetDebugger(), 
+          if (INTERRUPT_REQUESTED(GetDebugger(),
```

Comment:
Seems like you formatted the whole file, which is a good idea in general but not within the same commit as "real" changes. You could push a commit formatting it if you like.

You can use the script https://clang.llvm.org/docs/ClangFormat.html#script-for-patch-reformatting to format just the bits you've changed.

Reply:
BTW, I don't really agree that reformatting whole files is a great idea.  clang-format keeps changing it's mind on how it wants to lay out code, and so you just keep introducing little pointless changes that make history grubbing harder, etc, to very little benefit.